### PR TITLE
feat(ci): enable node test

### PR DESCRIPTION
**Pull request by leoryu**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind deprecation
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -170,39 +170,39 @@ jobs:
...
+          "
+            }
+          }
+          EOF
+          ) | curl '${{ secrets.WECHATWORK_WEBHOOK }}' \
+            -H 'Content-Type: application/json' \
+            -d @-
+          ginkgo -nodes=2 -v test/e2e_installer/node
 
   report-failure:
```
Comment (huxiaoliang) [inline .github/workflows/release.yml:207]: why keep 2 similar code block?  #L173-#L205 and new codes

**Review by huxiaoliang — CHANGES_REQUESTED** [review]
please address my comments

File: .github/workflows/release.yml (hunk 1 above)
Comment (leoryu) [reply .github/workflows/release.yml:207]: Here is no new code, I just uncommented old code.

**Comment by zhangzhangzf** [comment]
lgtm

**Review by zhangzhangzf — APPROVED** [review]
lgtm

**Pull request merged**